pkg/controller/node/node_controller_test.go

```diff
@@ -816,7 +816,7 @@ func TestPaused(t *testing.T) {
 	expMcp.Status = expStatus
 	f.expectUpdateMachineConfigPoolStatus(expMcp)
 
-	f.run(getKey(mcp, t))
+	f.runExpectError(getKey(mcp, t))
```

Comment:
should we now have a successful test where `f.run` just runs? (mocking machines len)